l10n_ec_partner/country.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014-Today Jonathan Finlay <jfinlay@riseup.net>.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import fields, osv
+
+class CountryCity(osv.osv):
+    _name = 'res.country.city'
+    _description = "City model"
+
+    _columns = {
+        'country_id': fields.many2one('res.country', 'Country',
+                                      required=True),
+        'state_id': fields.many2one('res.country.state', 'State',
+                                    domain="[('country_id','=',country_id)]",
+                                    required=True),
+        'name': fields.char('Name', size=64, required=True),
+        'code': fields.char('Code', size=4, required=True),
+    }
+
```

Comment:
Creería yo que l10n_ec_geo es más semántico

On Mon, Jul 27, 2015 at 12:23 PM, Cristian Salamea <notifications@github.com
> wrote:

> In l10n_ec_partner/country.py
> <https://github.com/odoo-ecuador/odoo-ecuador/pull/53#discussion_r35561833>
> :
>
> > +from openerp.osv import fields, osv
> > +
> > +class CountryCity(osv.osv):
> > +    _name = 'res.country.city'
> > +    _description = "City model"
> > +
> > +    _columns = {
> > +        'country_id': fields.many2one('res.country', 'Country',
> > +                                      required=True),
> > +        'state_id': fields.many2one('res.country.state', 'State',
> > +                                    domain="[('country_id','=',country_id)]",
> > +                                    required=True),
> > +        'name': fields.char('Name', size=64, required=True),
> > +        'code': fields.char('Code', size=4, required=True),
> > +    }
> > +
>
> Existen ya varios modulos para esta personalizacion, sin embargo por la
> arquitectura el modulo l10n_ec_partner es unicamente para personlizaciones
> respecto a esto.
>
> Seria mejor si lo movemos a un modulo: l10n_ec_country ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/odoo-ecuador/odoo-ecuador/pull/53/files#r35561833>.
>



-- 

*Dairon Medina CaroOpenSource Consultant*
*weblog:* http://codeadict.github.com
*mobile:* *(+593)*-987612278
*LinkedIn:* http://linkedin.com/in/codeadict
*GitHub:* https://github.com/codeadict
*Skype:* codeadict
*Twitter:* @codeadict